CFC_DataCollector/recommender/common.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import division
+import urllib2
+import xml.etree.ElementTree as ET
+from get_database import *
+from datetime import datetime, timedelta
+import trip
+from trip import E_Mission_Trip
+import random
+import json
+import jsonpickle
+
+def get_uuid_list():
+	uuid_list = [ ]
+	db = get_section_db()
+	for x in db.find():
+   		uuid_list.append(x['_id'])
+	return uuid_list
+
+def insert_into_pdb(pdb, my_id, new_perturbed_trip):
+	my_id = str(my_id)
+	assert(type(my_id) == str)
+	new_id = my_id.replace('.', "")
+	assert(type(new_id) == str)
+	if '.' not in new_id:
+		pdb.insert({'joh' : new_perturbed_trip})
```

Comment:
@joshzarrabi, what is `'joh'` and why is it hardcoded here?
Some comments on what you are doing here would be useful.
What does it mean if the new_id has a . or not?